CMakeLists.txt

```
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 3.22)
+
+project(ClpIrV1Decoder)
+
+# Download and extract Boost
+set(BOOST_VERSION "1.85.0")
```

Let's enable caching on the variable and also add a docstring for it.
```suggestion
set(BOOST_VERSION "1.85.0" CACHE STRING "Version of Boost library to be used when compiling CLP sources.")
```